README.md

```diff
@@ -26,6 +26,8 @@ The flags are:
 	regular expression for files to ignore
 -modtime=""
 	Unix timestamp to override as modification time for all files
+-private
+	Use non-exported function names. Standard FS* functions will be prefixed with esc.  Eg: FSString => escFSString.
```

Comment:
Use the same style as the other lines:

- no period at EOL or in the line, use a semicolon if needed
- no caps at start except for proper names

Maybe:

`unexport functions by prefixing them with esc; e.g., FS -> escFS`